### PR TITLE
Add react-testing-utils + Fix Analytics

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

adds react testing-library 

and a couple of tests which use it 

also add test to analytics and rewrite to ensure that A) we only load once, and B) `track` is never called until its loaded

### Tested
for analytics new test and in browser check too

` yarn test` for the rest

Screen Shot 2020-01-14 at 3 48 40 PM

### Other changes

adds a few labels and alt text  accessibility
fix a warning on `Link` about there not being a anchor tag

### CODE REVIEW

File: packages/web/src/brandkit/common/MobileMenu.test.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { fireEvent, getByTitle, render } from '@testing-library/react'
+import * as React from 'react'
+import MobileMenu from './MobileMenu'
+
+describe(MobileMenu, () => {
+  xdescribe('when pressed', () => {
```
Comment (jmrossy) [inline packages/web/src/brandkit/common/MobileMenu.test.tsx]: Did you mean to have this skipped? 
In general I recommend `describe.skip` b.c. it's less easy to miss

File: packages/web/src/privacy/Privacy.tsx [hunk 2]
```diff
@@ -284,7 +284,7 @@ export default class Privacy extends React.Component {
           <P>
             You may also contact us to address and resolve concerns you may have about our use of
             your Personal Data. Please contact us at{' '}
-            <Link href="mailto:privacy@celo.org">privacy@celo.org</Link>.
+            <Link href="mailto:privacy@celo.org"><a>privacy@celo.org</a></Link>.
```
Comment (jmrossy) [inline packages/web/src/privacy/Privacy.tsx:287]: It's strange to me that the Link component isn't responsible for wrapping the content in an anchor

**Review by jmrossy — COMMENTED** [review]
Looks good, just a few questions below

File: packages/web/src/privacy/Privacy.tsx (hunk 2 above)
Comment (aaronmgdr) [reply packages/web/src/privacy/Privacy.tsx:287]: one of the WTFisms of rn

File: packages/web/src/brandkit/common/MobileMenu.test.tsx (hunk 1 above)
Comment (aaronmgdr) [reply packages/web/src/brandkit/common/MobileMenu.test.tsx]: oh yes, good call.

Comment (aaronmgdr) [reply packages/web/src/brandkit/common/MobileMenu.test.tsx]: got the test working

**Review by aaronmgdr — COMMENTED** [review]
@jmrossy fixed that skipped test.  good to go?

**Comment by jmrossy** [comment]
@aaronmgdr looks like there's a conflict

**Comment by aaronmgdr** [comment]
@jmrossy im declaring this a conflict free zone 😉

**Review by jmrossy — APPROVED** [review]
✅

**Pull request merged**